Dockerfile

```
@@ -1,7 +1,12 @@
-FROM alpine:3.17
+FROM ubuntu:22.04
 
-# install packages required to run the tests
-RUN apk add --no-cache jq coreutils
+RUN apt-get update
+RUN apt-get install -y jq coreutils wget zip libfontconfig1
+# Download and unzip Godot v4.0.1
+RUN wget https://downloads.tuxfamily.org/godotengine/4.0.1/Godot_v4.0.1-stable_linux.x86_64.zip
+RUN unzip Godot_v4.0.1-stable_linux.x86_64.zip
+RUN mv Godot_v4.0.1-stable_linux.x86_64 /usr/bin/godot
+RUN rm Godot_v4.0.1-stable_linux.x86_64.zip
```

This also reduces the image size a bit

```suggestion
RUN wget https://downloads.tuxfamily.org/godotengine/4.0.1/Godot_v4.0.1-stable_linux.x86_64.zip && \
    unzip Godot_v4.0.1-stable_linux.x86_64.zip && \
    mv Godot_v4.0.1-stable_linux.x86_64 /usr/bin/godot && \
    rm Godot_v4.0.1-stable_linux.x86_64.zip
```

---

Changed, thanks!